polaris-react/src/components/PositionedOverlay/utilities/math.ts

```diff
@@ -117,7 +117,12 @@ export function calculateHorizontalPosition(
 export function rectIsOutsideOfRect(inner: Rect, outer: Rect) {
   const {center} = inner;
 
-  return center.y < outer.top || center.y > outer.top + outer.height;
+  return (
+    center.y < outer.top ||
+    center.y > outer.top + outer.height ||
+    inner.left < outer.left ||
+    inner.left + inner.width > outer.left + outer.width
```

Comment:
This closes the popover once it reaches the left or right side of the activator. This works with all alignments of the popover and I think looks nicer than closing when reaching the center of the activator

Reply:
going to remove this for now since I don't have a fix for the bulk editor :( we can easily add it back in a separate PR if needed though!